README.md

```diff
@@ -36,7 +36,7 @@ Testing
 Running Tests
 ----
 1. populate test database: `bundle exec rake wikirate:test:seed`
-1. start rspec with `decko rspec` and cucumber with `decko cucumber`.
+2. start rspec with `decko rspec`cucumber with `decko cucumber`.
```

Comment:
probably want the "and" back.

(fwiw, markdown handles ordered lists with _any_ number):

```
1. 1
5. 2
1. 3
```
=
1. 1
5. 2
1. 3